src/core/broadcaster.ts

```diff
@@ -55,10 +32,10 @@ export module Core {
      * If there is already a callback associated with that key, then the callback will be replaced.
      *
      * @param key The key associated with the callback. Key uniqueness is determined by deep equality.
-     * @param {BroadcasterCallback} callback A callback to be called when the Scale's domain changes.
-     * @returns {Broadcaster} this object
+     * @param {(listenable: L, ...args: any[]) => any} callback A callback to be called.
```

Comment:
I assume type doesn't work?  Maybe just use an interface here to better inform / encapsulate